BookPlayer/Services/BookSortService+SortError+PlayListSortOrder.swift

```diff
@@ -0,0 +1,35 @@
+import Foundation
+
+final class BookSortService {
```

Comment:
It could, but having sorting being handled by the model seems to be mixing concerns to me.

Reply:
I was thinking about this on the way to work and another possibility is to pass a playlist to the service and this would remove the need to have a sort method in the playlist itself. Let dependency injection handle the whole thing.